opm/input/eclipse/EclipseState/Tables/TableManager.hpp

```diff
@@ -420,6 +425,8 @@ namespace Opm {
         double m_rtemp {288.7056}; // 60 Fahrenheit in Kelvin
         double m_salinity {0.0};
         int m_actco2s {3};
+        int m_thco2mixbrine {3};
+        int m_thco2mixgas {0};
```

Comment:
Can we make these be `enum class` instead, please?  A bare integer makes it so much harder to reason about the potential states.